picatic.io/app/components/example-container/example-container.template.html

```diff
@@ -0,0 +1,17 @@
+<figure ng-class="{'col-md-offset-2': $first}" class="grid__brick col-6 col-sm-6 col-md-4 col-xl-4 shuffle-item shuffle-item--visible" ng-repeat="example in $ctrl.examples" ng-cloak>
```

Comment:
.col-6 and .col-xl-4 aren't actual classes.
Even if they were, they would be redundant redundancies.